src/schema/__tests__/fair.test.js

```diff
@@ -228,6 +231,23 @@ describe("Fair", () => {
     })
   })
 
+  it("includes a formatted exhibition period", async () => {
+    const query = gql`
+      {
+        fair(id: "aqua-art-miami-2018") {
+          exhibition_period
+        }
+      }
+    `
+
+    const data = await runQuery(query, rootValue)
+    expect(data).toEqual({
+      fair: {
+        exhibition_period: "Feb 15 – 17",
```

Comment:
@kierangillen This test is broken, because the function it uses (`exhibitionPeriod`) is incomplete. It doesn’t take a timezone parameter and thus in my timezone this ends up being rendered as `Feb 16 - 18`. The right fix here is to make `exhibitionPeriod` take a timezone and use `defaultTimezone` from the resolver context.

I'm disabling the test in my PR.

Reply:
I'll create a PR today to fix this test.